lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -953,6 +953,31 @@ static LogicalResult verify(spirv::BranchConditionalOp branchOp) {
 // spv.CompositeExtractOp
 //===----------------------------------------------------------------------===//
 
+static Type getElementType(Type type, ArrayAttr indices, Location loc) {
```

Comment:
Could you move this to the top of this file and put it under the "Common utility functions" section?

Reply:
Thansk, fixed!